include/seqan3/alignment/pairwise/detail/policy_alignment_matrix.hpp

```diff
@@ -92,10 +92,14 @@ class policy_alignment_matrix
         lower_diagonal = band.lower_diagonal.get();
         upper_diagonal = band.upper_diagonal.get();
 
-        if (upper_diagonal < lower_diagonal || upper_diagonal < 0 || lower_diagonal > 0)
+        bool invalid_band = upper_diagonal < lower_diagonal;
+        if constexpr (traits_t::is_global)
+            invalid_band |= (upper_diagonal < 0 || lower_diagonal > 0);
+
+        if (invalid_band)
             throw invalid_alignment_configuration{"The selected band [" + std::to_string(lower_diagonal) + ":" +
-                                                  std::to_string(upper_diagonal) + "] is not supported because it "
-                                                  "does not completely encloses both sequences."};
+                                                  std::to_string(upper_diagonal) + "] cannot be used with the current "
```

Comment:
Did I request this in another PR? please add a note why the band is invalid in the error message